src/v2/Components/FrameWithRecentlyViewed.tsx

```diff
@@ -1,20 +1,15 @@
-import { Box, Flex } from "@artsy/palette"
+import { Flex } from "@artsy/palette"
 import { HorizontalPadding } from "v2/Apps/Components/HorizontalPadding"
-import { useSystemContext } from "v2/Artsy"
 import React from "react"
 import { LazyLoadComponent } from "react-lazy-load-image-component"
 
-import { Footer } from "v2/Components/Footer"
 import { RecentlyViewedQueryRenderer as RecentlyViewed } from "v2/Components/RecentlyViewed"
 
 export interface Props {
   name?: string
 }
 
 export const FrameWithRecentlyViewed: React.SFC<Props> = ({ children }) => {
-  const { isEigen } = useSystemContext()
-  const showFooter = !isEigen
```

Comment:
This logic was added for a reason, but I wonder what that reason was / if its even needed anymore? Let me do some digging and check out the history... maybe we can just omit this for the sake of progress.

Reply:
I'm inclined to let this slide. This component renders a rail that is totally responsive. So if for whatever reason Eigen needs to render a webview, there will be a functional, mobile-friendly version of their recently viewed artworks ready for them to browse. 

So all good from my side 👍 In the future though, we'll need to be extra sure we're not removing logic that might be needed in some other context (such as `isEigen`), or if removed, at least call it out in a self-comment after the PR is opened.

Reply:
Nice catch!